zegami_sdk/collection.py

```diff
@@ -175,6 +177,19 @@ def rows(self):
             self._cached_rows = df
 
         return df
+    
+    @property
+    def tags():
+        pass
+
+    @tags.getter
+    def tags(self):
+        if self.allow_caching and self._cached_tags is not None:
+            return self._cached_tags
+        tags = self.get_tags()
+        if self.allow_caching:
+            self._cached_tags = tags
+        return tags
```

Comment:
Given that tags are intended to be a sort've dynamic column, I wonder if caching may be problematic compared to rows. If a user modifies the tags and doesn't clear the cache, they will get incorrect data.

Reply:
Is it better to not cache the tags? I was trying to mimic the same behaviour in js-sdk where tags are cached.